packages/api/scripts/protocgen.sh

```diff
@@ -11,7 +11,7 @@ for dir in $PROTO_DIRS; do
     -I "proto" \
     -I "third_party/proto" \
     --plugin=${TS_PROTO_BIN} \
-    --ts_proto_opt="esModuleInterop=true,forceLong=long,useOptionals=true,outputTypeRegistry=true" \
+    --ts_proto_opt="esModuleInterop=true,forceLong=long,useOptionals=messages,outputTypeRegistry=true" \
```

Comment:
What does this change do?

Reply:
using `true` has just been deprecated in favor of `messages`, see https://github.com/stephenh/ts-proto docs but it does the same thing